Source/WebCore/editing/cocoa/AttributedString.h

```diff
@@ -86,6 +86,108 @@ using AttributedStringTextTableBlockID = LegacyNullableObjectIdentifier<Attribut
 struct AttributedStringTextListIDType;
 using AttributedStringTextListID = LegacyNullableObjectIdentifier<AttributedStringTextListIDType>;
 
+enum class TextTableLayoutAlgorithm : uint8_t {
+    Automatic,
+    Fixed
+};
+
+enum class TextTableBlockVerticalAlignment : uint8_t {
+    Top,
+    Middle,
+    Bottom,
+    Baseline,
+};
+
+enum class ParagraphStyleAlignment : uint8_t {
+    Natural,
+    Left,
+    Right,
+    Centre,
+    Justified,
+};
+
+enum class ParagraphStyleWritingDirection: uint8_t {
+    Natural,
+    LeftToRight,
+    RightToLeft,
+};
+
+struct WEBCORE_EXPORT ParagraphStyleTextList {
+    AttributedStringTextListID thisID { };
+    String markerFormat { };
+    long startingItemNumber = 0;
```

Comment:
There's a recent thread in webkit-dev email list where we have decided to use { 0 } to initialize these instead of = 0